WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/NotificationsUtils.java

```diff
@@ -49,6 +50,7 @@
     public static final String WPCOM_PUSH_DEVICE_NOTIFICATION_SETTINGS = "wp_pref_notification_settings";
     private static final String WPCOM_PUSH_DEVICE_SERVER_ID = "wp_pref_notifications_server_id";
     public static final String WPCOM_PUSH_DEVICE_UUID = "wp_pref_notifications_uuid";
+    public static final String NOTICON_FONT_NAME = "Noticons-Regular.otf";
```

Comment:
Oh, yeah. I moved it because I was using it in two places at one point but forgot to move it back.